front/src/TurtleEnableButton.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+/**
+ * A button that keeps track whether a turtle is enabled or not.
+ * Is reflected to the user in terms in terms of button appearance.
+ *
+ * Props:
+ * - enabled: whether this turtle is enabled or not
+ * - onDisable: function to call when disabling this turtle
+ * - onEnable: function to call when enabling this turtle
+ *
+ * Author: S.A. Tanja
+ * Author: H.E. van der Laan
+ * Author: T.T.P. Franken
+ */
+const TurtleEnableButton = props => {
+  if (props.enabled) {
+    return (
+      <Button className={props} isActive onClick={props.onDisable}>
+        {props.id}
+      </Button>
+    );
+  } else {
+    return (
+      <Button className={props} onClick={props.onEnable}>
+        {props.id}
+      </Button>
+    );
+  }
+};
+
+TurtleEnableButton.propTypes = {
+  enabled: PropTypes.bool.isRequired,
+  onDisable: PropTypes.func.isRequired,
+  onEnable: PropTypes.func.isRequired
+};
+
+const Button = styled.button`
+  border-style: ${props => (props.isActive ? "inset" : "solid")};
+  border-color: ${props => (props.isActive ? "none" : "#ededed")};
+  background-color: ${props => (props.isActive ? "silver" : "#ededed")};
+  width: 16%;
+  min-width: 75px;
+  height: 10vh;
+  font-size: 4vmin;
+  flex: 1;
+  margin: 1px;
```

Comment:
Use `rem` instead of `vmin` as well (bodgan commented this on `NotificationWindow.js`).